docker/data/logstash/env2yaml/env2yaml.go

```diff
@@ -94,12 +94,10 @@ func normalizeSetting(setting string) (string, error) {
 		"modules",
 		"path.logs",
 		"path.plugins",
-		"password_policy.mode",
-		"password_policy.length.minimum",
-		"password_policy.include.upper",
-		"password_policy.include.lower",
-		"password_policy.include.digit",
-		"password_policy.include.symbol",
+		"api.auth.type",
```

Comment:
@yaauie it seems AUTH related options weren't added. I wonder if adding is safe. In case I am running docker & windows acceptance integration tests.

Reply:
[Docker tests](https://logstash-ci.elastic.co/job/elastic+logstash+main+multijob-docker-acceptance/90/) and [Windows tests](https://logstash-ci.elastic.co/job/elastic+logstash+main+multijob-windows-compatibility/) seem good.

Reply:
:+1: it looks like we missed that when adding the feature. Good catch to add them in here.